docs/04-operation-guides/operations/01-install-kyma.md

```diff
@@ -80,9 +81,11 @@ Optionally, you can specify from which source you want to deploy Kyma. For examp
 For example, to install Kyma from a specific version, such as `1.19.1`, run:
 
   ```bash
-  kyma deploy --source=1.19.1
+  kyma deploy --source=2.0
   ```
 
+>**NOTE:** You can choose to write either `kyma deploy --source=2.0` or `kyma deploy --source=release-2.0`.
```

Comment:
I would rewrite this note, in a way that it is clear for the user that `release-2.0` references a specific branch in the GitHub repository. And `2.0` references the release.
Eventho, we are sticking to the "for each release one branch" strategy, I think we should tell the user.

I.e. if a branch called `test-branch` exists in the upstream kyma repo, it is also possible to do `kyma deploy --source=test-branch`